content/achievements.md

```diff
@@ -19,6 +19,19 @@ These ones were quite successful:
     <h2 class="timelineMajorMarker"><span>2017</span></h2>
     <dl class="timelineMinor">
 
+      <dt id="event32"><a>RoboCup Junior World Championship Nagoya</a></dt>
+      <dd class="timelineEvent" id="event32EX" style="display: none; ">
+        <div><ul>
+             <li>Soccer Lightweight Primary Individual games - 12th place</li>
```

Comment:
@vargac For the consistency with previous years, please remove "Individual games"